web/views.py

```diff
@@ -292,7 +292,7 @@ def dispatch(self, request, *args, **kwargs):
             return super().dispatch(request)
         else:
             return HttpResponseForbidden(
-                _("The study %s is not currently collecting data - the study is either completed or paused. If you think this is an error, please contact %s")%(study.name,study.contact_info)
+                _(f"The study {study.name} is not currently collecting data - the study is either completed or paused. If you think this is an error, please contact {study.contact_info}")
             )
```

Comment:
If I am remembering correctly, I ran into this issue while I was working on recent feature.  I ended breaking up the string using concatenation and translating the "unchanging" parts of the string.  For example:
```
_("The study ") + study.name + _("is not currently collecting data - the study is either completed or paused. If you think this is an error, please contact") + study.contact_info
```
I understand the variable won't be translated in the above version, but maybe you guys have a preference.  I could go either way, I just want to be consistant and make the lives of the translators are easy as possible.

Reply:
Sometimes the location of the inserted component shifts between languages, and if someone just translates "The study" and "is not currently collecting..." then the concatenated result might not make sense. I'm not sure about this specific sentence, but we had this problem in another location with Japanese recently. It is probably less error prone to translate whole sentences, as long as our translators are well-schooled in not translating the {study.name} tag.

Reply:
I think going forward, we'll stick to your "method" of translating text.  I'll am going to put in a issue to confirm that all translated text doesn't take on my "incorrect concatenation" I mentioned above.   

https://github.com/lookit/lookit-api/issues/840